.travis.yml

```diff
@@ -57,7 +57,7 @@ matrix:
 
 before_script:
 # Init PHP
-  - export CORE_RELEASE=$TRAVIS_BRANCH
```

Comment:
why has this been removed? mistake or 2 fixes in 1?

Reply:
Because CORE_RELEASE is a relic of travis-support-module and isn't necessary anywhere.